subworkflows/local/utils_nfcore_crisprseq_pipeline/main.nf

```diff
@@ -474,12 +472,12 @@ def validateParametersScreening() {
         warning "mle_design_matrix will only be used for the MAGeCK MLE computations"
     }
 
-    if(params.fasta && params.count_table) {
-        error "Please provide either a fasta file or a count_table"
+    if(params.bowtie && params.count_table) {
+        warning "The count_table file will be used, bypassing the Bowtie alignment step"
```

Comment:
I would keep it as an error and not a warning, this might heavily confuse a user if they get results and might bias the rest of the analysis